sdk/core/azure-core/src/main/java/com/azure/core/http/policy/FixedDelayOptions.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http.policy;
+
+import com.azure.core.util.logging.ClientLogger;
+
+import java.time.Duration;
+import java.util.Objects;
+
+/**
+ * The configuration for exponential backoff.
+ */
+public class FixedDelayOptions {
+    private Integer maxRetries;
+    private Duration delay;
+
+    /**
+     * Gets the max retry attempts that can be made.
+     *
+     * @return The max retry attempts that can be made.
+     */
+    public Integer getMaxRetries() {
+        return maxRetries;
+    }
+
+    /**
+     * Sets the max retry attempts that can be made.
+     *
+     * @param maxRetries the max retry attempts that can be made.
+     *
+     * @return The updated {@link FixedDelayOptions}
+     */
+    public FixedDelayOptions setMaxRetries(Integer maxRetries) {
+        Objects.requireNonNull(maxRetries, "'maxRetries' cannot be null.");
+        if (maxRetries < 0) {
+            ClientLogger logger = new ClientLogger(FixedDelayOptions.class);
```

Comment:
can you please do a static logger instead? It improves startup perf (100x) by eliminating env var lookup